packages/@n8n/config/src/configs/credentials.config.ts

```diff
@@ -16,10 +16,6 @@ class CredentialsOverwrite {
 
 @Config
 export class CredentialsConfig {
-	/** Default name for credentials */
-	@Env('CREDENTIALS_DEFAULT_NAME')
```

Comment:
should we document this removal as a "breaking change"?

Reply:
Is it tho? It only affect the behaviour of the internal API and we don't make any promises for that?

Reply:
we (unfortunately) have it publicly documented [here](https://docs.n8n.io/hosting/configuration/environment-variables/credentials/).

Reply:
In that case we can't break this until in the next major. I'll revert the change